model/pipeline/pom.xml

```diff
@@ -46,6 +46,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>${maven-checkstyle-plugin.version}</version>
```

Comment:
Ditto

Reply:
All should go in `<pluginManagement/>` (even using `<reporting/>`).